### PR TITLE
doc(postfix): install instructions

### LINKED ISSUE #6131: postfix plugin: update README with installation instructions
and also indicate that the plugin as it currently stands is meant for developer use/testing.

### PR DESCRIPTION
**Pull request by sydneyli**
fixes #6131

### CODE REVIEW

File: certbot-postfix/README.rst [hunk 1]
```diff
@@ -2,9 +2,26 @@
...
 
-``certbot install --installer postfix --cert-path <path to cert> --key-path <path to key> -d <MX hostname>``
+To install this plugin, in the root of this repo, run::
 
-And there you go! If you'd like to obtain these certificates via certbot, there's more documentation on how to do this `here <https://certbot.eff.org/docs/using.html#getting-certificates-and-choosing-plugins>`_.
+    ./tools/venv.sh
+    source venv/bin/activate
+
+You can use this installer with any `authenticator plugin
+<https://certbot.eff.org/docs/using.html#getting-certificates-and-choosing-plugins>_`.
```
Comment (bmw) [inline certbot-postfix/README.rst]: The underscore should be after the closing backtick.

File: certbot-postfix/README.rst [hunk 2]
```diff
@@ -2,9 +2,26 @@
...
+For instance, with the `standalone authenticator
+<https://certbot.eff.org/docs/using.html#standalone>_`, which requires no extra server
```
Comment (bmw) [inline certbot-postfix/README.rst]: The underscore should be after the closing backtick.

File: certbot-postfix/README.rst [hunk 3]
```diff
@@ -2,9 +2,26 @@
...
+software, you might run::
+
+    certbot run --standalone -i postfix -d <domain name>
+
+To just install existing certs with this plugin, run::
+
+    certbot install -i postfix --cert-path <path to cert> --key-path <path to key> -d <domain name>
+
+If you need elevated permissions to run ``certbot``, you can run
+``sudo ./venv/bin/certbot`` with the above arguments.
```
Comment (bmw) [inline certbot-postfix/README.rst]: This is just a suggestion which you can ignore if you want, but the user will almost certainly need elevated permissions to run Certbot. We need these permissions to:

* Access Certbot's default directories
* Bind to port 80 for standalone
* Modify Postfix config files at the default path
* Run Postfix utilities

With this in mind, perhaps we should change these instructions to use `sudo` above which makes things easier for most users. There is value in giving people the option to run Certbot without root, but it requires a lot of additional setup which this doc doesn't get into.

Comment (sydneyli) [reply certbot-postfix/README.rst]: Good point-- Also, I'm pretty sure you need `sudo` to be able to reload `postfix` at all.

**Comment by sydneyli** [comment]
@bmw thanks for the review. on a vaguely related note :notes:, I just noticed and am amused by my commit message typo :musical_score:

**Pull request merged**